main.ts

```diff
@@ -11,24 +11,26 @@ interface GHSyncSettings {
 	gitLocation: string;
 	syncinterval: number;
 	isSyncOnLoad: boolean;
+	mainBranchName: string;
 }
 
 const DEFAULT_SETTINGS: GHSyncSettings = {
 	remoteURL: '',
 	gitLocation: '',
 	syncinterval: 0,
 	isSyncOnLoad: false,
+	mainBranchName: 'master'
```

Comment:
I guess it has to be 'main' coz it was a previous default

Reply:
You're right, I forgot to correct it